src/main/webapp/queue.js

```diff
@@ -128,7 +150,7 @@ async function getRoomQueue(roomid) {
   fetch('/queueRefresh?roomid=' + roomid)
       .then((response) => response.json())
       .then((queue) => {
-        if (queue != null) {
+        if (queue != null && queue != undefined && queue.length > 0) {
```

Comment:
what's the difference between null and undefined in JS?

Reply:
null is an object whereas undefined is a type, I check for both to be safe but I can eliminate one